packages/web-components/src/components/in-page-banner/in-page-banner.scss

```diff
@@ -7,8 +7,9 @@
 
 @use '@carbon/ibmdotcom-styles/scss/globals/vars' as *;
 @use '@carbon/ibmdotcom-styles/scss/internal/content-block';
-@use '@carbon/ibmdotcom-styles/scss/components/content-group-banner';
+// @use '@carbon/ibmdotcom-styles/scss/components/in-page-banner';
+@use '../../../../styles/scss/components/in-page-banner';
 
-:host(#{$c4d-prefix}-content-group-banner) {
+:host(#{$c4d-prefix}-in-page-banner) {
   display: block;
 }
```

Comment:
if we move the styles to the styles package, then we can just have 
`@use '@carbon/ibmdotcom-styles/scss/components/in-page-banner';`

Reply:
oh whoops yep